src/libtorch.cc

```diff
@@ -1221,23 +1221,25 @@ ModelInstanceState::ProcessRequests(
   // synchronized the stream in the ReadOutputTensors function.
   if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
 #ifdef TRITON_ENABLE_GPU
+    // [FIXME] in the case of cudaEventElapsedTime failure, should handle
```

Comment:
I was wondering was this the reason for intermittent segfault after the synchronization has been fixed? `cudaEventElapsedTime` should not fail since we synchronize the stream before reaching this part of the code.

Reply:
It's hard to tell, I have not yet been able to catch anything after the synchronization fix (segfault less often?). But note that in general CUDA API may return error from unrelated operation ("Note that this function may also return error codes from previous, asynchronous launches.")